### DISCUSSION TITLE
v1.3.0

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#3567] [Announcements]
## Windows App SDK 1.3.0 (1.3.230331000)

WinAppSDK 1.3.0 is the latest stable version of WinAppSDK providing new features and improvements to the platform. This release focuses on improving how you develop with WinAppSDK.

**New features include:**
- Easier to debug your app through improved error handling: New event for failed XAML resource lookup​.
- Simpler to use APIs for existing features: Window.AppWindow (custom title bar) & Window.SystemBackdrop (Mica)​.

**Additional updates include:**
- Numerous performance, security, accessibility and reliability fixes to improve the quality of your app.
- Updated WinUI Gallery, Template Studio, and VS WinUI templates.

**To see everything that's new and changed, see the full [Windows App SDK 1.3 release notes](https://aka.ms/windowsappsdk/1.3/1.3.0/release-notes).**  

**See the [WinAppSDK 1.3 milestone](https://github.com/microsoft/WindowsAppSDK/milestone/14?closed=1) and [WinUI 3 in WinAppSDK 1.3 milestone](https://github.com/microsoft/microsoft-ui-xaml/milestone/18?closed=1) for a list of issues addressed in this release.**

### Try it out
- Download the [1.3.230331000 NuGet](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.3.230331000) package to use WinAppSDK 1.3 in your app.
- Download and update the [WinUI Gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC) to see the WinUI 3 updates firsthand.

> Of note, the API reference docs for this release temporarily say "1.3 Preview", this will be fixed shortly.

### Getting Started 

To get started using Windows App SDK to develop Windows apps, check out the following documentation:    
- [Install developer tools ](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).     
- [Create a WinUI 3 app](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app).   
- [Continue your development journey](https://docs.microsoft.com/windows/apps/develop).   

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.3.0'>v1.3.0</a>.</em>

**Comment by Balkoth** [comment]
Why report bugs and track them internal when they get ignored and make it into the final release like this one: https://github.com/microsoft/WindowsAppSDK/issues/3572#issue-1667718027

**Reply by bpulliam** [reply]
This issue was fixed in 1.3.1.

**Comment by ghost1372** [comment]
@gabbybilka  vsix is not updated for v1.3!
still using v1.2
there is no Windows App SDK C# VS2022.vsix in marketplace and removing and reinstalling WASDK from visual studio installer still using v1.2

**Comment by bpulliam** [comment]
We found an issue with 1.3.0 VSIX and are fixing that for 1.3.1. The update will ship with 1.3.1.

**Reply by ghost1372** [reply]
@gabbybilka  @bpulliam  Still no updates! Can you put the download link of the new version? Everything is installed through Visual Studio and there is no update.
v1.3.1 is released.

**Reply by bpulliam** [reply]
I created issue #3635 to properly track this and to make our comments and updates more discoverable. We are working to get by these unforseen issues that are preventing their upgrade.

**Comment by wbokkers** [comment]
Tried 1.3 with our preview users. We see many more crashes happening with 1.3 than with 1.2
(14% of our users versus 0.1% of our users)
I can't reproduce these crashes myself, but I see them happening in App Center.

**Reply by gabbybilka** [reply]
@bpulliam @Scottj1s Can you follow up?

**Comment by Marv51** [comment]
@gabbybilka Looks like you shipped the nuget for 1.3.1 on tuesday? What is the status of the release notes? I see @bpulliam post that quite a few fixes made it into 1.3.1?

**Reply by gabbybilka** [reply]
Yep, they're here: https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-131-13230502000
